components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1470,6 +1471,8 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
                     ...(await hostContext.services.repositoryService.getRepositoriesForAutomatedPrebuilds(user, {
                         searchString: params.searchString,
                         cancellationToken,
+                        limit: params.limit,
+                        cap: params.maxPages,
```

Comment:
`cap` felt a bit confusing to expose as an argument on the rpc method so went with `maxPages`. I didn't want to change the `cap` parameter since it was in place already, but would be happy to rename it to `maxPages` consistently through the 2 layers it's passed.

Reply:
I think we should introduce another parameter then. They do have different meanings.

Reply:
Introduce a new parameter where? On the rpc method (I introduced `maxPages` there), or in the bitbucket server api impl (currently has `cap`). Having a `maxRequests` param would probably be the most accurate, but it sure feels odd to expose that for callers to pick how many underlying requests we make against the scm provider. Thoughts?

Reply:
I meant your original suggestion `cap` to `maxPages`. But It is not a blocker if you want to land.